### PR TITLE
DSL2 module refactor

### PR DESCRIPTION
**Pull request by tilschaef**
- Updated README and template.nf with new DSL2 syntax.

### CODE REVIEW

File: Utils/template.nf [hunk 1]
```diff
@@ -6,10 +6,11 @@ process Command {
     shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple sample_id, file(input_file)
-
+    path input_file
+
```
Comment (rernst) [inline Utils/template.nf:10]: A better example would be something like this?
```
tuple sample_id, file(input_file)
file(second_input_file)
```

**Comment by rernst** [comment]
I think we should make sure to adjust all modules as well, not just the Readme? Maybe create or rename this branch and make sure that we update all modules?

File: Utils/template.nf (hunk 1 above)
Comment (tilschaef) [reply Utils/template.nf:10]: Agreed but it appears that file is now replaced with path (see DSL2 docs)

**Comment by tilschaef** [comment]
Sure, I renamed the branch.

**Pull request closed without merging**